Examples/instance-flow.md

```diff
@@ -1,61 +1,105 @@
+
+## Obtain Deployment Variables
 
-**Declare Deployment Variables**
+Next, define the variables that will contain the parameters of the script. This way, each time you run the script, you can pass different deployment arguments. These declared variables are referenced later in your script as request parameter values.
 
-In this case we decided to use AWS so we will need the account key and secret to register it as provider. Box deployment arguments such as owner, environment and token are declared as variables because they can differ between deployments. That way, each time you run the script, you can pass different arguments.
+| Variable | Description |
+|----------|-------------|
+| provider_key | You need the provider account key and secret to register a provider. |
+| provider_secret | See below how to [register a provider](#Register-a-Provider-in-Cloud-Application-Manager) and get its parameters. |
+| json_web_token | This is the [authentication token](#Authenticate-with-Cloud-Application-Manager) obtained before.<br>It's a long BASE64 encoded string that starts with something like:<br/>  *eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJvcGVyYb25z...* |
+| environment | Set your URL organization hostname or your appliance IP, where API calls are sent.<br>Examples are:  *your-organization.cam.ctl.io*, *centurylink.cam.ctl.io*,  *192.168.1.10*. |
+| owner | Specify the user ID in Cloud Application Manager who owns the provider account. |
+
+Start with your script with the following lines:
 ```
-aws_provider_key=$1
-aws_provider_secret=$2
-environment=$3
-json_web_token=$4
+# Deployment common variables
+provider_key=$1
+provider_secret=$2
+json_web_token=$3
+environment=$4
 owner=$5
 ```
 
-### Register a Provider in Cloud Application Manager
+There are other variables obtained during the script execution and necessary in next script steps.
+| Variable | Description |
+|----------|-------------|
+| provider_id | Obtained when registering a provider. |
+| policy_box_id | Obtained when creating a deployment policy box. |
+| box_id | Obtained when creating a custom box. |
+| instance_id | Obtained when deployin a box instance. |
+
+
+## Register a Provider in Cloud Application Manager
+
+A provider is a public or private cloud account you can register in Cloud Application Manager. 
 
-A provider is a public or private cloud account you can register in Cloud Application Manager. In this case we choose AWS as provider. From the AWS account credential we will need the key and the secret right now.
+To register a provider send a POST request to the [Providers endpoint](https://www.ctl.io/api-docs/cam/#cam-platform-providers-api) with the requiered parameters.
 
+In this case choose create an AWS as provider in an appliance. It will be necessary a key and the secret pair.  
```

Comment:
_This example follows the steps to create an AWS provider in a CAM appliance._

_...organization..._

_...you can obtain check in..._ > _...you can check in..._